packages/nextjs/app/builders/0x21dfd1CfD1d45801f46B0F40Aed056b064045aA2/page.tsx

```diff
@@ -0,0 +1,147 @@
+"use client";
+
+import React from "react";
+import Link from "next/link";
+
+export default function RoheemahBuilderPage() {
+  return (
+    <div className="min-h-screen flex flex-col md:flex-row items-top justify-start gap-16 px-4 md:px-20">
+      <div className="flex-1">
+        <div className="flex flex-col md:flex-row items-start justify-start gap-14 mt-20">
+          <div className="relative">
+            <div className="absolute top-0 left-0 w-full h-full bg-[#9BF9F3] rounded-tl-[100px] rounded-br-[100px] -z-10 transform -translate-x-[4px] -translate-y-[4px]"></div>
+            <div className="leaf-shaped py-8 px-6 md:px-10 border-2 flex flex-col items-left justify-start bg-[#0D1117] rounded-tl-[100px] rounded-br-[100px] relative z-10">
+              <div className="flex flex-col items-center justify-start">
+                <div className="w-24 h-24 rounded-full bg-cover bg-center bg-no-repeat border-2 border-[#6EF4E6] text-[20px] font-bold text-white flex items-center justify-center">
+                  A.R.A
+                </div>
+                <p className="text-[24px] font-normal">Roheemah</p>
+                <p className="text-md text-[#9B9EA1]">Frontend & Blockchain Dev</p>
+              </div>
+              <div className="mt-6 text-[#9B9EA1]">
```

Comment:
Inside this `<div>` there is a lot of receptive code, maybe you could refactor it?